corehq/apps/hqcase/api/get_list.py

```diff
@@ -128,7 +138,9 @@ def _get_query(domain, params):
              .sort("_uid", reset_sort=False))
     for key, val in params.lists():
         if len(val) == 1:
-            query = query.filter(_get_filter(domain, key, val[0]))
+            filter = _get_filter(domain, key, val[0])
+            if filter:
```

Comment:
FYI an alternative to this is to have the filter return `filters.match_all()` instead of `None` when you want it to no-op.  No strong preference either way.

Reply:
Thanks for the suggestion. I prefer the idea of returning something than having the function conditionally return `None`, as this makes the function a little bit more consistent. I have implemented this where necessary so that we don't need the `if filter` conditional check anymore.